docusaurus/docs/reactnative/basics/troubleshooting.mdx

```diff
@@ -295,10 +295,22 @@ public class MainActivity extends ReactActivity {
 }
 ```
 
-## Custom touchable not working
+## Touchables not working
```

Comment:
I was just scrolling through this file, and apparently we already have a section called "Touchables Not Working". We should merge those two. That section is about v1 of RNGH. We can just include instructions about v2 there and leave this section as "Custom Touchable Not Working"

Reply:
Done 😄